sdk/sqlmanagement/Azure.ResourceManager.Sql/README.md

```diff
@@ -0,0 +1,77 @@
+# Azure SQL Management client library for .NET
+
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+
+## Getting started 
+
+### Install the package
+
+Install the Azure SQL management library for .NET with [NuGet](https://www.nuget.org/):
+
+```PowerShell
+Install-Package Azure.ResourceManager.Sql -Version 1.0.0-preview.1 
+```
+
+### Prerequisites
+
+* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+
+### Authenticate the Client
+
+To create an authenticated client and start interacting with Azure resources, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
```

Comment:
We should create an AKA link for this.

Reply:
and many of the URIs in this doc

Reply:
We will do a full pass of all of the readme at the end.